site/server/app/com/fortysevendeg/exercises/persistence/PersistenceImpl.scala

```diff
@@ -0,0 +1,59 @@
+package com.fortysevendeg.exercises.persistence
+
+import doobie.imports._
+import shapeless.HNil
+
+import scala.language.higherKinds
+import scalaz.Foldable
+
+class PersistenceImpl {
```

Comment:
I believe @dialelo is about to merge too a bunch of related work to persistence. Have you guys coordinated this effort?